### DISCUSSION TITLE
Customtkinter problem

### DISCUSSION THREAD

**Discussion by jonathan-muina19** [TomSchimansky/CustomTkinter#2497] [Q&A]
How can I ensure that the widgets, images and windows of my customtkinter python application can fit on all computers?

**Comment by Akascape** [comment]
@jonathan-muina19 
Use pack or grid method. If you are using pack then use expand and fill options. If you use grid then learn about the columnconfigure and rowconfigure methods, and also the sticky parameter.

After making the UI, test it on different screens, if you face any DPI scale issue then try these methods:

```python
import ctypes
ctypes.windll.shcore.SetProcessDpiAwareness(0)
```

or

```python
customtkinter.set_widget_scaling(1.0)
customtkinter.set_window_scaling(1.0)
customtkinter.deactivate_automatic_dpi_awareness()
```
